src/tezos/tezosPlugin.js

```diff
@@ -43,6 +43,23 @@ export class TezosPlugin extends CurrencyPlugin {
     }
   }
 
+  async importPrivateKey (userInput: string): Promise<Object> {
+    try {
+      const keys = eztz.crypto.generateKeys(userInput, '')
+      this.derivePublicKey({
+        type: 'wallet:tezos',
+        id: 'fake',
+        keys
+      })
+      return {
+        mnemonic: keys.mnemonic,
+        privateKey: keys.sk
+      }
+    } catch (e) {
+      throw new Error('Invalid input')
```

Comment:
`catch () { throw ... }` is completely useless - it throws away the *actual* error and replaces it with something that has less information. Just let the original eztz method throw whatever it wants.